src/handlers/payout/action.ts

```diff
@@ -51,10 +51,11 @@ export interface IncentivesCalculationResult {
 export interface RewardByUser {
   account: string;
   priceInEth: Decimal;
-  userId: string | undefined;
+  userId: number | undefined;
```

Comment:
`-1` would be more explicit to say that there was an error in getting the `userId`? Why should we support `undefined`?